src/shared/bus-util.c

```diff
@@ -724,8 +724,12 @@ int bus_print_property(const char *name, sd_bus_message *property, bool value, b
                 if (r < 0)
                         return r;
 
-                if (all || !isempty(s))
-                        print_property(name, "%s", s);
+                if (all || !isempty(s)) {
+                        bool good;
+
+                        good = !strchr(s, '\n');
```

Comment:
I think there should be a comment here, explaining why we look for `\n` here, and nothing else (for example, `=`?).

Also maybe turn the check into a static inline mini-function?